<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>751</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Menú Principal</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>441</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Laboratorio #4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>651</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>230</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>MENÚ</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ejercicio #1</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>441</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sistema de Gestión de Tarea</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>651</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="TE_Descripcion">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>150</y>
        <width>351</width>
        <height>221</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>110</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Descripción:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tipo de Tarea:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="tareaTipo">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Diaria</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Semanal</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Fecha:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDateEdit" name="DateEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>390</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>CREAR</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Ejercicio2">
     <attribute name="title">
      <string>Ejercicio #2</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>491</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sistema de Figuras Geométricas</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>651</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tipo de Figura</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>270</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>DIBUJAR</string>
      </property>
     </widget>
     <widget class="QComboBox" name="figuraTipo">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Círculo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rectángulo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Triángulo</string>
       </property>
      </item>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>140</y>
        <width>391</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="canvasWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Ejercicio #3</string>
     </attribute>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>491</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Editor de Texto con Formato</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>651</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="TE_Editor">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>491</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="negritaButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Negrita</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cursivaButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cursiva</string>
      </property>
     </widget>
     <widget class="QPushButton" name="subrayadoButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>320</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Subrayado</string>
      </property>
     </widget>
     <widget class="QPushButton" name="normalButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Normal</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Ejercicio #4</string>
     </attribute>
     <widget class="Line" name="line_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>651</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sistema de Gestión de Inventario</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="tipoProducto">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>80</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Electrónico</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ropa</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Alimento</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tipo de Producto:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>280</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Precio:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_NombreProducto">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="agregarBoton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>380</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>AGREGAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="LabelP">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>170</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_Adicional">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>210</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_Adicional2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>310</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="LabelS">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>280</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Precio:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="LE_PrecioProducto">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>310</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>Inventario</string>
     </attribute>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Inventario:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="eliminarBoton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>10</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>ELIMINAR</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>661</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Lista de Tareas</string>
     </attribute>
     <widget class="QListWidget" name="taskList">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>671</width>
        <height>381</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Lista de Tareas:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
